spec/team_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'team'
+require 'player'
+
+describe Team do
```

Comment:
Not bad, though personally I would have a single interface, `#valid?`, for knowing whether a team is valid or not, rather than two different methods arising from different criteria. After all, it's the responsibility of the Team class to know of the various rules that make a team valid.